grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -57,12 +57,12 @@ class ApiController {
 
     @CacheFlush(["megamenuCache"])
     def chooseLocale = {
-        Locale locale = localizationService.getLocale(params.id)
-        if (!locale) {
+        Locale newLocale = localizationService.getLocale(params.id)
```

Comment:
This seems unnecessary.

Reply:
what do you mean as unnecessary here? I'm a bit confused, because I only changed the naming from `locale` to `newLocale` and I'm not sure if changing of name seems unnecessary or this part: `localizationService.getLocale(params.id)`

Reply:
I meant changing the variable name isn't necessary and I want to avoid unnecessary changes as much as possible since it complicates the rebase process for OBGM.